### ISSUE TITLE
Error building Android apk with flutter beta 1.25.0-8.1.pre

### ISSUE THREAD

**Issue by nstrelow** [MaikuB/flutter_local_notifications#950]
**Incompatibility of android libraries**

**To Reproduce (possibly)**
1. Go to beta channel
2. Build apk with flutter local notifications

I can provide more details if needed.

**Error log**

```
❯ flutter build apk
Warning: You are using these overridden dependencies:
! intl 0.17.0-nullsafety.2
Running "flutter pub get" in app...                              2.612ms
You are building a fat APK that includes binaries for android-arm, android-arm64, android-x64.
If you are deploying the app to the Play Store, it's recommended to use app bundles or split the APK to reduce the APK size.
    To generate an app bundle, run:
        flutter build appbundle --target-platform android-arm,android-arm64,android-x64
        Learn more: https://developer.android.com/guide/app-bundle
    To split the APKs per ABI, run:
        flutter build apk --target-platform android-arm,android-arm64,android-x64 --split-per-abi
        Learn more: https://developer.android.com/studio/build/configure-apk-splits#configure-abi-split

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':flutter_local_notifications:compileReleaseAidl'.
> Could not resolve all task dependencies for configuration ':flutter_local_notifications:releaseCompileClasspath'.
   > Could not resolve androidx.lifecycle:lifecycle-runtime:{strictly 2.0.0}.
     Required by:
         project :flutter_local_notifications
      > Cannot find a version of 'androidx.lifecycle:lifecycle-runtime' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-runtime:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.lifecycle:lifecycle-runtime:2.1.0'

   > Could not resolve androidx.collection:collection:{strictly 1.0.0}.
     Required by:
         project :flutter_local_notifications
      > Cannot find a version of 'androidx.collection:collection' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.media:media:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.collection:collection:{strictly 1.0.0}' because of the following reason: releaseRuntimeClasspath uses version 1.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.versionedparcelable:versionedparcelable:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.collection:collection:1.1.0'

   > Could not resolve androidx.lifecycle:lifecycle-common:{strictly 2.0.0}.
     Required by:
         project :flutter_local_notifications
      > Cannot find a version of 'androidx.lifecycle:lifecycle-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common-java8:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'

   > Could not resolve androidx.arch.core:core-common:{strictly 2.0.0}.
     Required by:
         project :flutter_local_notifications
      > Cannot find a version of 'androidx.arch.core:core-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.arch.core:core-common:2.1.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.arch.core:core-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.arch.core:core-common:2.0.1'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-runtime:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'

   > Could not resolve androidx.lifecycle:lifecycle-runtime:2.0.0.
     Required by:
         project :flutter_local_notifications > androidx.core:core:1.3.0
      > Cannot find a version of 'androidx.lifecycle:lifecycle-runtime' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-runtime:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.lifecycle:lifecycle-runtime:2.1.0'

   > Could not resolve androidx.collection:collection:1.0.0.
     Required by:
         project :flutter_local_notifications > androidx.media:media:1.1.0
         project :flutter_local_notifications > androidx.versionedparcelable:versionedparcelable:1.1.0
      > Cannot find a version of 'androidx.collection:collection' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.media:media:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.collection:collection:{strictly 1.0.0}' because of the following reason: releaseRuntimeClasspath uses version 1.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.versionedparcelable:versionedparcelable:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.collection:collection:1.1.0'

   > Could not resolve androidx.lifecycle:lifecycle-common:2.2.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3
         project :flutter_local_notifications > androidx.lifecycle:lifecycle-runtime:2.2.0
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.lifecycle:lifecycle-common-java8:2.2.0
      > Cannot find a version of 'androidx.lifecycle:lifecycle-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common-java8:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'

   > Could not resolve androidx.lifecycle:lifecycle-runtime:2.2.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3
      > Cannot find a version of 'androidx.lifecycle:lifecycle-runtime' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-runtime:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.lifecycle:lifecycle-runtime:2.1.0'

   > Could not resolve androidx.arch.core:core-common:2.1.0.
     Required by:
         project :flutter_local_notifications > androidx.lifecycle:lifecycle-runtime:2.2.0
      > Cannot find a version of 'androidx.arch.core:core-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.arch.core:core-common:2.1.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.arch.core:core-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.arch.core:core-common:2.0.1'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-runtime:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'

   > Could not resolve androidx.collection:collection:1.1.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0
      > Cannot find a version of 'androidx.collection:collection' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.media:media:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.collection:collection:{strictly 1.0.0}' because of the following reason: releaseRuntimeClasspath uses version 1.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.versionedparcelable:versionedparcelable:1.1.0' --> 'androidx.collection:collection:1.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.collection:collection:1.1.0'

   > Could not resolve androidx.lifecycle:lifecycle-runtime:2.1.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.activity:activity:1.0.0
      > Cannot find a version of 'androidx.lifecycle:lifecycle-runtime' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.0.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-runtime:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.lifecycle:lifecycle-runtime:2.1.0'

   > Could not resolve androidx.arch.core:core-common:2.0.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.loader:loader:1.0.0 > androidx.lifecycle:lifecycle-livedata:2.0.0
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.loader:loader:1.0.0 > androidx.lifecycle:lifecycle-livedata:2.0.0 > androidx.arch.core:core-runtime:2.0.0
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.loader:loader:1.0.0 > androidx.lifecycle:lifecycle-livedata:2.0.0 > androidx.lifecycle:lifecycle-livedata-core:2.0.0
      > Cannot find a version of 'androidx.arch.core:core-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.arch.core:core-common:2.1.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.arch.core:core-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.arch.core:core-common:2.0.1'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-runtime:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'

   > Could not resolve androidx.arch.core:core-common:2.0.1.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.activity:activity:1.0.0 > androidx.savedstate:savedstate:1.0.0
      > Cannot find a version of 'androidx.arch.core:core-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.arch.core:core-common:2.1.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.arch.core:core-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.arch.core:core-common:2.0.1'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.arch.core:core-runtime:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.arch.core:core-common:2.0.0'

   > Could not resolve androidx.lifecycle:lifecycle-common:2.0.0.
     Required by:
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.activity:activity:1.0.0 > androidx.savedstate:savedstate:1.0.0
         project :flutter_local_notifications > io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3 > androidx.fragment:fragment:1.1.0 > androidx.loader:loader:1.0.0 > androidx.lifecycle:lifecycle-livedata:2.0.0 > androidx.lifecycle:lifecycle-livedata-core:2.0.0
      > Cannot find a version of 'androidx.lifecycle:lifecycle-common' that satisfies the version constraints:
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Constraint path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.lifecycle:lifecycle-common:{strictly 2.0.0}' because of the following reason: releaseRuntimeClasspath uses version 2.0.0
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'androidx.core:core:1.3.0' --> 'androidx.lifecycle:lifecycle-runtime:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.lifecycle:lifecycle-common-java8:2.2.0' --> 'androidx.lifecycle:lifecycle-common:2.2.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.activity:activity:1.0.0' --> 'androidx.savedstate:savedstate:1.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'
           Dependency path 'com.dexterous.flutterlocalnotifications:flutter_local_notifications:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_release:1.0.0-92ae191c17a53144bf4d62f3863c110be08e3fd3' --> 'androidx.fragment:fragment:1.1.0' --> 'androidx.loader:loader:1.0.0' --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' --> 'androidx.lifecycle:lifecycle-livedata-core:2.0.0' --> 'androidx.lifecycle:lifecycle-common:2.0.0'


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
Running Gradle task 'assembleRelease'...
Running Gradle task 'assembleRelease'... Done                      10,0s
The build failed likely due to AndroidX incompatibilities in a plugin. The tool is about to try using Jetifier to solve the incompatibility.
Building plugin connectivity...
Running Gradle task 'assembleAarRelease'...
Running Gradle task 'assembleAarRelease'... Done                    4,5s

C:\Users\nilss\AppData\Local\Pub\Cache\hosted\pub.dartlang.org\connectivity-0.4.9+5\android\src\main\java\io\flutter\plugins\connectivity\ConnectivityBroadcastReceiver.java:16: error: cannot find symbol
import androidx.annotation.RequiresApi;
                          ^
  symbol:   class RequiresApi
  location: package androidx.annotation
C:\Users\nilss\AppData\Local\Pub\Cache\hosted\pub.dartlang.org\connectivity-0.4.9+5\android\src\main\java\io\flutter\plugins\connectivity\ConnectivityBroadcastReceiver.java:66: error: cannot find symbol
  @RequiresApi(api = Build.VERSION_CODES.N)
   ^
  symbol:   class RequiresApi
  location: class ConnectivityBroadcastReceiver
2 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s


The plugin connectivity could not be built due to the issue above.
```

**Comment by MaikuB** [comment]
Seeing issues with this on other plugins so I've filed an issue on the main Flutter repository at https://github.com/flutter/flutter/issues/72627

**Comment by nstrelow** [comment]
Should be fixed now on `Flutter 1.25.0-8.2.pre`
